### ISSUE TITLE
Images not loading

### ISSUE THREAD

**Issue by TheHenn86** [HashLips/hashlips_nft_minting_dapp#45]
Hi I replaced the images you told me to in the read me and they are not showing up! Please Help

**Comment by Captbadger** [comment]
did you solve this

**Comment by TheHenn86** [comment]
Yes I solved the problem I renamed the images to the image names I replaced
Thanks for getting back to me it's appreciated.

**Comment by Captbadger** [comment]
Where did you do this , I’m have the same issue - meta dat shows, but not pictures.

**Comment by TheHenn86** [comment]
I dragged the images into studio visual code and then deleted the old
images and renamed the images I put in to the names of the old images and
that's how I solved this problem.

**Comment by Captbadger** [comment]
Cheers, I think my issue is different. If I mint from contract the image and properties show. If I mint from minting drop on the properties show on OS.
